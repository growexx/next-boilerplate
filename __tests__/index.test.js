import { render, screen } from '@testing-library/react'
import Home from '@src/pages'
import FeaturePage from 'page-components/FeaturePage'

describe('Home', () => {
  it('renders a heading', () => {
    render(<Home />)

    const heading = screen.getByRole('heading', {
      name: /welcome to next\.js!/i
    })

    expect(heading).toBeInTheDocument()
  })
})

describe('<FeaturePage />', () => {
  it('should render its heading', () => {
    const {
      container: { firstChild }
    } = render(<FeaturePage />)

    expect(firstChild).toMatchSnapshot()
  })
})
