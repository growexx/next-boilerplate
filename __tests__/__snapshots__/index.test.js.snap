// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FeaturePage /> should render its heading 1`] = `
<div
  class="sc-beySbM kyCJAz"
>
  [object Object]
</div>
`;
